.changeset/rare-grapes-explode.md

```diff
@@ -0,0 +1,7 @@
+---
+"openapi-typescript-helpers": minor
```

Comment:
Since these are all < 1.0, we want to only bump `minor` versions on breaking changes. We ship bugfixes and features on patches while on 0.x.

```suggestion
"openapi-typescript-helpers": patch
```